cmd/secret/cmd/root.go

```diff
@@ -63,14 +74,24 @@ func New(opts ...RootOptions) *root {
 		Long:    "Create CLI to set and get secrets via the command line",
 		Version: options.version,
 	}
+	path := []string{"secret.log"}
+
+	cfg := zap.NewDevelopmentConfig()
+	cfg.OutputPaths = path
+	cfg.ErrorOutputPaths = path
+	logg, err := cfg.Build()
+	if err != nil {
+		return nil
+	}
 
-	rootData := &root{cmd: secret, options: options}
+	rootData := &root{cmd: secret, options: options, logger: logg.Sugar()}
 
 	secret.AddCommand(rootData.setCmd())
 	secret.AddCommand(rootData.getCmd())
 	secret.AddCommand(rootData.serverCmd())
 	secret.SilenceUsage = true // write false if you want to see options when an error occurs
 
+	secret.Flags().StringVarP(&logFile, "log-file", "l", "go-secret/cmd/secret/cmd", "path to log file")
```

Comment:
Quite unnecessary details on default path of logfile. Better to use local path, for example`./`